driver-core/src/test/java/com/datastax/driver/core/policies/AbstractRetryPolicyIntegrationTest.java

```diff
@@ -44,10 +44,10 @@
 
 /**
  * Base class for retry policy integration tests.
- * <p/>
+ * <p>
```

Comment:
ack, my bad, my environment is targeting JDK 8 which uses `<p>` instead of `<p/>` will fix.